pkg/store/README.md

```diff
@@ -0,0 +1,91 @@
+---
```

Comment:
I don't think we need this - metadata is meant for Swarm documentation page (user guide), but `pkg/` won't be included in it (I think?)